src/main/webapp/js/form.js

```diff
@@ -1,6 +1,31 @@
+$("#randomize-form").submit(function(event) {

```

Comment:
Is this a different form? It is above the comment below that says that the 3P code is for handling the form on RRP

Reply:
oops, didn't label this function. it is the same form as the comment below refers to.